controller/state/statedb.go

```diff
@@ -8,12 +8,14 @@ import (
 	"errors"
 	"fmt"
 	"io"
+	"io/ioutil"
 	"net/http"
 	"sync"
 	"time"
 
 	"github.com/filecoin-project/dealbot/metrics"
 	"github.com/filecoin-project/dealbot/tasks"
+	blockformat "github.com/ipfs/go-block-format"
```

Comment:
non-blocking: for better or worse, in most repos the shortname used here is 'blocks'